.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Mark or close stale issues and PRs'
+
+on:
+    schedule:
+        - cron: '00 * * * *'
```

Comment:
I'm not that familiar with cron syntax... is this once an hour, at the top of the hour?

Reply:
@mattsherman no this is once a day at "minute 0". I think reading up on cron tab patterns is an exercise for you if you're interested: https://www.netiq.com/documentation/cloud-manager-2-5/ncm-reference/data/bexyssf.html

I'll try summarise though:

cron goes: minute, hour, day(month), month, day(week).

the * means any value so this will run once at minute 0 every day.

You can also use this website to confirm how a value works: https://crontab.guru/#00_*_*_*_*